test/lib/java-wrapper.test.ts

```diff
@@ -20,3 +24,9 @@ test('not target folder throw error', async () => {
     getTargets('some-bogus-folder-that-does-not-exist'),
   ).rejects.toThrowError('Could not find a target folder');
 });
+
```

Comment:
How about a test checking that the file was written properly?

Reply:
What would be the benefit of this test? It sounds to me like testing node api for writing files. However, one more test making sure the file is removed properly makes sense.

Reply:
But it will be part of e2e one.